### ISSUE TITLE
application

### ISSUE THREAD

**Issue by Vcting** [bargavjayaraman/secure_model_aggregation#3]
Hello, I would like to ask you the following questions:
1. How is the corresponding parameter in the experiment set? What is the specific amount?
2. What are the application scenarios of this model besides hospitals?
3. When running a real dataset, there is a significant gap between the results of the last two methods. Is it the reason for the parameters?
4. How to compare the privacy of the four methods?
I look forward to your reply, thank you very much.

**Comment by bargavj** [comment]
Hi! Are you talking about the results in Table 1 of the paper (https://pmpml.github.io/PMPML16/papers/PMPML16_paper_15.pdf)? 
1. In secure_model_aggregation.h, you can change the values of M (number of parties) and D (dimensions). Apart from that the other model training parameters are handled in the python code. The paper mentions what model hyper-parameters we set.
2. This framework could be used in any setting where multiple parties wish to compute a function over their private inputs without revealing the inputs to any parties.
3. There is always some loss in model accuracy when we move from centralized to distributed setting when we aggregate the model parameters as we do in this work. Since, each party has less data to train their private models on, the individual models are less accurate.
4. Only the last method (Our Approach in Table 1) is private. The other three methods are non-private.
